<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trackersCollecterClass</class>
 <widget class="QWidget" name="trackersCollecterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trackersCollecter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <widget class="QTableWidget" name="listTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>380</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>180</number>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>From</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="title_collectedTrackers">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Collected Trackers</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="val_collectedTrackers">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 32pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copy">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_state">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editLocalTrackers">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Local Trackers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editOnlineSources">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Online Sources</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="trackersCollecter.qrc"/>
 </resources>
 <connections/>
</ui>
